### ISSUE TITLE
2.0 Single & Mult-Select + Dropdown {Component}

### ISSUE THREAD

**Issue by Jaibles** [calcom/cal.com#3856]
[Select in Figma](https://www.figma.com/file/9MOufQNLtdkpnDucmNX10R/%E2%9D%96-Cal-DS-v2.0?node-id=385%3A33491) | [Select Dropdown in Figma](https://www.figma.com/file/9MOufQNLtdkpnDucmNX10R/%E2%9D%96-Cal-DS-v2.0?node-id=1388%3A67631)

![Image](https://user-images.githubusercontent.com/8019099/185884315-3678abdf-da1d-4bb2-8876-0e197ff7919a.png)

**Comment by alishaz-polymath** [comment]
Redesigning the mult-select dropdown/select according to the figma designs requires quite a bit of overhead. Thoughts on using https://www.primefaces.org/primereact/multiselect/ for that purpose instead? Looks like a decent solution that meets our needs from the looks of it. If we're not against the idea of using this, I can test it out locally to see how it works, otherwise will keep looking for the solutions in `react-select` 🙏 
CC: @PeerRich @Jaibles

**Comment by Jaibles** [comment]
I'm open to discussing changes to the designs if there is something we can do that is supported that is reasonable and meets the same need. But the prime demo also looks good. However, I'm definitely not the one to make the call on whether using something like this is a good call in the long run.

**Comment by alishaz-polymath** [comment]
> I'm open to discussing changes to the designs if there is something we can do that is supported that is reasonable and meets the same need. We don't need the custom chip with avatar for example. Let's just drop it.
> 
> But the prime demo also looks good. However, I'm definitely not the one to make the call on whether using something like this is a good call in the long run.

Yep, makes sense. I think that if we are to keep react select for now, maybe we can make changes to the design for the interim period where we make it look more part of the V2 theme for the moment, and then we can give it the time it needs to convert it to the final design as a follow up. What do you think?

**Comment by Jaibles** [comment]
Sounds good to me.

**Comment by alishaz-polymath** [comment]
> The "2 more" part tricky?

It is, yeah @Jaibles. Also, I checked through quick changes that the cross button with each name isn't quite working well either, although this could simply be a minor bug, but a bug nonetheless. 
I mean I would think that it is best to accommodate it the way it is right now and just make it not feel out of place from the design. And then someone works at it for the final design (ps. I could take it up as well). I just feel we might not have the time, unless there's someone who can take it up before then.

**Comment by alishaz-polymath** [comment]
Tldr; I would definitely aim to have it done as per the final designs, but I don't know if we have the time (unless someone can confirm that they do).

**Comment by Jaibles** [comment]
> cross button
Do you mean the remove? Could we just allow removing by opening the dropdown and clicking again to deselect as a temporary fix?

**Comment by alishaz-polymath** [comment]
> Do you mean the remove icon? Could we just allow removing by opening the dropdown and clicking again to deselect as a temporary fix?

Yeah, so the remove icon for each user doesn't work, the main remove icon doesn't work either (can't click). I'll have a go at it with the suggested changes, and see if I can get it to work in a partial state, report back with my findings 🙏
